src/tests/test_bz2.py

```diff
@@ -7,6 +7,6 @@ def test_bz2():
 
     text = "Hello test test test test this is a test test test"
     some_compressed_bytes = bz2.compress(text.encode("utf-8"))
-    assert some_compressed_bytes != text
+    assert some_compressed_bytes != text.encode("utf-8")
```

Comment:
This is likely a mistake, as otherwise it's comparing str to bytes, which is always not true.